test/functional/host_test.rb

```diff
@@ -312,6 +312,21 @@
 
     assert_cmd(expected_result, result)
   end
+
+  it 'should create a host with default owner-type User' do
+
+    api_expects(:hosts, :create, 'Create a host with an owner').with_params(
+      'location_id' => 1, 'organization_id' => 1, 'host' => {
+         'name' => 'test'
+      }
+    ).returns(results: { 'owner_type' => 'User', 'name' => 'test' })
+
+    expected_result = success_result("Host created.\n")
+
+    result = run_cmd(cmd + minimal_params_without_hostgroup)
```

Comment:
As far as I see we don't pass `--owner` option here, so `onwer_id` is not being called, thus the test doesn't actually tests the functionality of that method.